cloud/aws-functions/orphaned_openshift_instances.py

```diff
@@ -0,0 +1,59 @@
+# Remove openshift resources.
+import logging
+import datetime
+import boto3
+import sys
+from time import sleep
+from botocore.exceptions import ClientError
+from boto3.exceptions import Boto3Error
+
+
+def isInstanceToTerminate(instance):
+    tags = instance.tags
+    tags_dict = {item['Key']: item['Value'] for item in tags}
+    state = instance.state['Name']
+    if 'team' not in tags_dict.keys() or state != 'running':
+        return False
+    if 'delete-cluster-after-hours' not in tags_dict.keys() and tags_dict['team'] == 'cloud':
+        return True
+
+    instance_lifetime = float(tags_dict['delete-cluster-after-hours'])
+    current_time = datetime.datetime.now().timestamp()
+    creation_time = instance.launch_time.timestamp()
+
+    if (current_time - creation_time) / 3600 > instance_lifetime and tags_dict['team'] == 'cloud':
+        return True
+    return False
+
+def get_instances_to_terminate(aws_region):
+    instances_for_deletion = []
+    ec2 = boto3.resource('ec2', region_name=aws_region)
+    instances = ec2.instances.all()
+    if not instances:
+        logging.info(f"There are no instances in cloud")
+        sys.exit("There are no instances in cloud")
+    for instance in instances:
+        if isResourceToTerminate(instance):
+            instances_for_deletion.append(instance.id)
+
+    if not instances_for_deletion:
+        logging.info(f"There are no instances for deletion")
+        sys.exit("There are instances for deletion")
+    return instances_for_deletion
+
+def delete_instance(aws_region, instance_id):
+    ec2 = boto3.resource('ec2', region_name=aws_region)
+    ec2.instances.filter(InstanceIds = [instance_id]).terminate()
```

Comment:
Don’t use spaces around the = sign when used to indicate a keyword argument. See [PEP8](https://peps.python.org/pep-0008/#other-recommendations)

```suggestion
    ec2.instances.filter(InstanceIds=[instance_id]).terminate()
```